website/views.py

```diff
@@ -368,3 +368,13 @@ def resolve_guid(guid, suffix=None):
 
     # GUID not found
     raise HTTPError(http.NOT_FOUND)
+
+# redirect ###
+
+
+def redirect_about(**kwargs):
+    return redirect('/4znzp/wiki/home/')
```

Comment:
Since this depends on the prod database, it would make sense to use the full URL (`http://osf.io/....`) here.